imagepullsecret-patcher/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# build stage
```

Comment:
It seems like the whole application repo has been put in the helm chart. That seems like a counterintuitive location. Is there any reason not to have our own repo or fork if we need to build the docker image ourselves?

Reply:
Was wondering about that.  Maybe now is the time to figure out the best way to have the helm chart code live within the same repo as the app and reference it from the devops repo like we do for loki/kubecost.

Reply:
Good call. We should have a meeting to think about our options.

Reply:
I'll do that and drop this PR.